docs/tutorials/data-modeling-lifecycle/part-2-analytic-solution.md

```diff
@@ -55,22 +65,23 @@ Olav starts by setting up the metadata for the solution model. He opens the `Met
 and fills in the following information:
 
 
-|             |                                                       |
-|-------------|-------------------------------------------------------|
-| role        | information architect                                 |
-| creator     | Olav                                                  |
-| namespace   | http://purl.org/cognite/power_analytic                |
-| prefix      | power_analytic                                        |
-| schema      | extended                                              |
-| created     | 2024-03-26                                            |
-| updated     | 2024-02-09                                            |
-| version     | 0.1.0                                                 |
-| title       | Power Forecast Model                                  |
-| description | Solution model for WindFarm power production forecast |
-
-The most important part of the metadata sheet is the `prefix` and `schema`. The `prefix` is used differentiate
-the solution model from the enterprise model. If Olav set the `prefix` to the same as the enterprise model, this
-would mean that this is an extension of the enterprise model and not a new model. The `schema` is used to
+|               |                                                       |
+|---------------|-------------------------------------------------------|
+| role          | information architect                                 |
+| creator       | Olav                                                  |
+| dataModelType | solution                                              |
+| namespace     | http://purl.org/cognite/power_analytic                |
```

Comment:
```suggestion
| namespace     | http://purl.org/cognite/power_analytic/                |
```

the namespace must end with either "/" or "#" , I think we do magically fix this (at least we were doing this in legacy)